woocommerce/woocommerce
# Updating all instances of now-deprecated `pnpx` to `pnpm dlx`

### All Submissions:


### Changes proposed in this Pull Request:

The usage of `pnpx` has a utility has been deprecated in favor of using `pnpm dlx` and `pnpm exec`, as shown by [this issue](https://github.com/pnpm/pnpm/pull/3652) and [the documentation](https://pnpm.io/6.x/pnpx-cli). This PR is intended to update all references. 


_Note: Auto formatting made some broader changes to whitespace in a few of the files. They should align with configured rules, but I can revert those changes if need be as well_

Closes #32508 .

### How to test the changes in this Pull Request:

1. Checkout branch
2. Run `pnpm install && pnpm nx composer-install woocommerce` 
3. Run `pnpm nx build woocommerce`
4. No errors should be encountered, and all just build successfully.
5. Might be best to smoke test other build processes touched by the changes.

### Other information:


### FOR PR REVIEWER ONLY:
